assets/js/app.js

```diff
@@ -0,0 +1,73 @@
+String.prototype.shuffle = function () {
+  var a = this.split(""),
+    n = a.length;
+
+  for (var i = n - 1; i > 0; i--) {
+    var j = Math.floor(Math.random() * (i + 1));
+    var tmp = a[i];
+    a[i] = a[j];
+    a[j] = tmp;
+  }
+  return a.join("");
+}
+
+function generatePassword() {
+
+  let passwordLength = parseInt(document.querySelector("#inputPasswordLength").value);
+  if (!(passwordLength == NaN) && (passwordLength >= 8) && (passwordLength <= 128)) {
+
+    let charset = "abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ"
```

Comment:
If the option of numbers is only clicked then the charset will still have lower case and upper case numbers. It should be only the options that are clicked